### ISSUE TITLE
Highlight unread conversations

### ISSUE THREAD

**Issue by ZenImagine** [jez500/Private-Message-Messenger#8]
![capture du 2018-02-27 12-16-58](https://user-images.githubusercontent.com/29839861/36725917-26c19a10-1bb8-11e8-9759-c626e2b118f9.png)
In this example, I received a new message. I see 1 on the notification logo but I do not know what conversation is new.

I would need to highlight the unread conversations or put an indicator on that one

**Comment by Pls** [comment]
Yep, this is a problem. When there is more than one conversation, you have to guess which one has new messages. Would be nice to just have class "unread" on that thread to indicate unreads separately.

Patches or guides welcome if someone already managed to do this feature ;)

**Comment by jez500** [comment]
I thought this worked? at least for the recent messages block. If not it will in the near future
